packages/blade/src/components/Alert/Alert.stories.tsx

```diff
@@ -34,11 +35,51 @@ const Page = (): ReactElement => {
         Alerts are messages that communicate information to users about any significant changes or
         explanations inside the system in a prominent way.
       </Subtitle>
+      <FigmaEmbed title="Alert Figma Designs" src={figmaURL} />
       <Title>Usage</Title>
-      <Link withArrow={true} href={figmaURL} target="_blank" rel="noreferrer noopener">
-        View in Figma
-      </Link>
-      <Highlight language="tsx">{`import { Alert } from '@razorpay/blade/components' \nimport type { AlertProps } from '@razorpay/blade/components'`}</Highlight>
+      <Sandbox editorHeight={500}>
+        {`
+        import { useState } from 'react';
+        import { Alert, Button } from '@razorpay/blade/components';
+
+        function App() {
```

Comment:
Just showing the alert should be fine, shouldn't it (without a button to toggle) 🤔

Reply:
Was trying to make it a bit more similar to realistic use case. In applications people would want to show alert when some action fails or something happens right?

Reply:
Yes, that works. Though alerts can also appear without any user interaction. For example your paid plan has expired and you visit the dashboard. Non blocker.

Reply:
Yup that also makes sense. Will merge for now. We can see how users are using the alerts and other components and improve the examples based on that later.